File: Net/OnlineStore.Domain/ImageInfo.cs
namespace OnlineStore.Domain
{
    public class ImageInfo
    {
        public string Path { get; private set; }

        public ImageInfo(string path)
        {
            this.Path = path;
        }

        public string ImageType
        {
            get
            {
                return this.Path.Substring(this.Path.IndexOf(".") + 1);
            }
        }
    }
}

File: ContinuousIntegration_AOLV/src/Java/src/main/java/domain/ImageInfo.java
package domain;

public class ImageInfo {

	public String path;

	public ImageInfo(String path) {
		this.path = path;
	}

	public String getPath() {
		return path;
	}

	/**
	 * @return the type of the image
	 */
	public String getImageType() {
		return path.substring(path.indexOf(".") + 1);

	}
}
